homeassistant/components/aladdin_connect/__init__.py

```diff
@@ -1 +1,54 @@
 """The aladdin_connect component."""
+
+import logging
+from typing import Final
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.COVER]
+
+_LOGGER: Final = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: config_entries.ConfigEntry
+) -> bool:
+    """Set up platform from a ConfigEntry."""
+    hass.data.setdefault(DOMAIN, {})
+    hass_data = dict(entry.data)
+    # Registers update listener to update config entry when options are updated.
+    # unsub_options_update_listener = entry.add_update_listener(options_update_listener)
+    # Store a reference to the unsubscribe function to cleanup if an entry is unloaded.
+    # hass_data["unsub_options_update_listener"] = unsub_options_update_listener
+    hass.data[DOMAIN][entry.entry_id] = hass_data
```

Comment:
There is no value in adding your entry data to hass data
```suggestion
```

Reply:
Appears there is value, as it is read back when the cover.py entity is executed (as I learned when it was removed).  This is where the username/password lives.